inventory-fusion/src/test/scala/com/normation/inventory/provisioning/fusion/TestSignatureService.scala

```diff
@@ -0,0 +1,178 @@
+/*
+*************************************************************************************
+* Copyright 2015 Normation SAS
+*************************************************************************************
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* In accordance with the terms of section 7 (7. Additional Terms.) of
+* the GNU Affero GPL v3, the copyright holders add the following
+* Additional permissions:
+* Notwithstanding to the terms of section 5 (5. Conveying Modified Source
+* Versions) and 6 (6. Conveying Non-Source Forms.) of the GNU Affero GPL v3
+* licence, when you create a Related Module, this Related Module is
+* not considered as a part of the work and may be distributed under the
+* license agreement of your choice.
+* A "Related Module" means a set of sources files including their
+* documentation that, without modification of the Source Code, enables
+* supplementary functions or services in addition to those offered by
+* the Software.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Affero General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with this program. If not, see <http://www.gnu.org/licenses/agpl.html>.
+*
+*************************************************************************************
+*/
+
+
+package com.normation.inventory.provisioning.fusion
+
+import org.junit.runner._
+import org.specs2.mutable._
+import org.specs2.runner._
+import com.normation.utils.StringUuidGeneratorImpl
+import net.liftweb.common._
+import com.normation.inventory.domain.InventoryReport
+import com.normation.inventory.services.provisioning._
+import java.security.PublicKey
+import com.normation.inventory.domain.KeyStatus
+import java.security.Security
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+
+
+@RunWith(classOf[JUnitRunner])
+class TestSignatureService extends Specification with Loggable {
+
+  Security.addProvider(new BouncyCastleProvider());
+  private[this] def getInputStream (path : String) = {
+
+      import java.net.URL
+      val url = this.getClass.getClassLoader.getResource(path)
+
+      if(null == url) throw new NullPointerException(s"Resource with relative path '${path}' is null (missing resource? Spelling? Permissions?)")
+      url.openStream()
+  }
+
+  private[this] implicit class TestParser(parser: FusionReportUnmarshaller) {
+    def parse(reportRelativePath: String): InventoryReport = {
+      val is = getInputStream(reportRelativePath)
+
+      logger.info(is)
+      val r  = parser.fromXml("report", is)
+      logger.info(reportRelativePath)
```

Comment:
To be more precise: it will output a lot of things when running test on builds, and it does not seems necessary